### ISSUE TITLE
Validation # PSNR: inf

### ISSUE THREAD

**Issue by yaolugithub** [xinntao/BasicSR#136]
Hi,xintao.
   I trained the ESRGAN codes with DIV2K800 training set and DIV2K100 validation set.
When the training iterates to 125,000 times（epoch 121）, the PSNR is inf. After that, the problem of INF appeared intermittently,such as 140000,150000 iterations. Besides, PSNR is stable at around 31.1 when it is not inf.

I think it is abnormal training state when psnr is inf.  how can i recognize  the noramal training state? To see l_g_pix , l_g_fea,l_g_gan,l_d_real,l_d_fake,D_real,D_fake?  What is the noraml range of above values?  thanks.

**Comment by xinntao** [comment]
It should not be Inf. 

A log file for your reference. 
[train_190201-171143.log](https://github.com/xinntao/BasicSR/files/3227968/train_190201-171143.log)

**Comment by yaolugithub** [comment]
Thanks a lot.  could you send me a train log about ESRGAN since you gave me the SRGAN log?

**Comment by xinntao** [comment]
@yaolugithub 
I cannot find an ESRGAN's log. They are very similar I think.
